subgraphs/euler-finance/src/mappings/helpers.ts

```diff
@@ -217,53 +214,37 @@ export function createLiquidation(event: Liquidation): BigDecimal {
   return liquidation.amountUSD;
 }
 
-function updateMarketLendingFactors(marketUtility: _MarketUtility, usdcMarketUtility: _MarketUtility): void {
+export function updateLendingFactors(event: GovSetAssetConfig): void {
+  const marketUtility = getOrCreateMarketUtility(event.params.underlying.toHexString());
+  marketUtility.borrowFactor = event.params.newConfig.borrowFactor;
+  marketUtility.collateralFactor = event.params.newConfig.collateralFactor;
+  marketUtility.save();
+
   /**
    * Euler has different collateral and borrow factors for all assets. This means that, in theory, there
    * would be maximumLTV and collateralThreshold for every possible asset pair.
    * 
-   * For the sake of simplicity when calculating maximumLTV and liquidationThreshold,
-   * let's assume borrowed asset is always USDC and collateral asset is always given asset.
+   * For the sake of simplicity:
+   *  The maximumLTV is the collateral factor. This is the risk-adjusted liquidity of assets in a market
+   *  The liquidationThreshold is the borrow factor. If the borrow goes over the borrow factor 
+   *                times the collateral factor of the collateral's market the borrow is at 
+   *                risk of liquidation.
    *
-   * maximumLTV = usdcBorrowFactor * assetCollateralFactor 
-   * liquidationThreshold = maximumLTV
+   * maximumLTV = collateralFactor 
+   * liquidationThreshold = borrowFactor
    */
-  const market = getOrCreateMarket(marketUtility.id);
-
-  const marketCollateralFactor = marketUtility.collateralFactor.toBigDecimal().div(CONFIG_FACTOR_SCALE);
-  const usdcBorrowFactorDecimal = usdcMarketUtility.borrowFactor.toBigDecimal().div(CONFIG_FACTOR_SCALE);
-
-  market.maximumLTV = marketCollateralFactor.times(usdcBorrowFactorDecimal).times(BigDecimal.fromString('100'));
-  market.liquidationThreshold = market.maximumLTV;
-  if (market.maximumLTV != BIGDECIMAL_ZERO) {
-    market.canUseAsCollateral = true;
-  }
+  let market = getOrCreateMarket(marketUtility.id);
+  market.maximumLTV = marketUtility
+    .collateralFactor
+    .toBigDecimal()
+    .div(CONFIG_FACTOR_SCALE);
+  market.liquidationThreshold = marketUtility
+    .borrowFactor
```

Comment:
As per your comment, shouldn't the liquidation threshold be `collateralFactor * borrowFactor`?

Or well, we don't know the borrowFactor, since it would be the one from the asset being borrowed, which might not match the one from this market.

Would it make sense to make it then `collateralFactor * min(borrowFactor among all markets)`?
Or `collateralFactor * max(borrowFactor among all markets)`.
Or `collateralFactor * avg(borrowFactor among all markets)`, depending on the accuracy we want.